Longest_Increasing_Subsequence/Longest_Increasing_Subsequence.kt

```diff
@@ -0,0 +1,51 @@
+/* Kotlin code for Longest increasing subsequence/*
+
+class LIS
+{
+    fun lis(int:arr[], int:n):int
+    {
+        int lis[] = new int[n];
+        int result = Integer.MIN_VALUE;
+
+        for (i in 0 until n)
+	{
+	    lis[i] = 1;
+	}
+            
+	for (i in 1 until n)
+        {
+            for (j in 0 until i)
+            {
+                if (arr[i] > arr[j] && lis[i] < lis[j] + 1)
+		{
+		     lis[i] = lis[j] + 1;
+		}
+	    }
+            result = Math.max(result, lis[i]);
+        }
+
+        return result;
+    }
+
+    fun main()
+    {
+        var read = Scanner(System.`in`)
+    	println("Enter the size of Array:")
```

Comment:
fix indentation

Reply:
Please fix indentation here, still not fixed.

Reply:
you didn't do this change yet